src/main/java/slimeknights/mantle/command/BookCommand.java

```diff
@@ -0,0 +1,183 @@
+import slimeknights.mantle.Mantle;
+import slimeknights.mantle.client.book.BookLoader;
+import slimeknights.mantle.client.book.data.BookData;
+import slimeknights.mantle.client.screen.book.BookScreen;
+import slimeknights.mantle.network.packet.OpenNamedBookPacket;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/** A command for different book */
+public class BookCommand {
+  private static final String EXPORT_SUCCESS = "command.mantle.book.export.success";
+  private static final String EXPORT_FAIL = "command.mantle.book.export.error_generic";
+  private static final String EXPORT_FAIL_IO = "command.mantle.book.export.error_io";
+
+  /**
+   * Registers this sub command with the root command
+   * @param subCommand  Command builder
+   */
+  public static void register(LiteralArgumentBuilder<CommandSourceStack> subCommand) {
+    subCommand.requires(source -> source.hasPermission(MantleCommand.PERMISSION_GAME_COMMANDS) && source.getEntity() instanceof AbstractClientPlayer)
+      .then(Commands.literal("open")
+        .then(Commands.argument("id", ResourceLocationArgument.id()).suggests(MantleClientCommand.REGISTERED_BOOKS)
+          .executes(BookCommand::openBook)))
+      .then(Commands.literal("export_images")
+        .then(Commands.argument("id", ResourceLocationArgument.id()).suggests(MantleClientCommand.REGISTERED_BOOKS)
+          .executes(BookCommand::exportImages)));
+  }
+
+  /**
+   * Opens the specified book
+   * @param context  Command context
+   * @return  Integer return
+   */
+  private static int openBook(CommandContext<CommandSourceStack> context) {
+    ResourceLocation book = ResourceLocationArgument.getId(context, "id");
+
+    BookData bookData = BookLoader.getBook(book);
+    if(bookData != null) {
+      bookData.openGui(Component.literal("Book"), "", null, null);
+    } else {
+      OpenNamedBookPacket.ClientOnly.errorStatus(book);
+      return 1;
+    }
+
+    return 0;
+  }
+
+  /**
+   * Renders all images in the book to files
+   * @param context  Command context
+   * @return  Integer return
+   */
+  private static int exportImages(CommandContext<CommandSourceStack> context) {
+    ResourceLocation book = ResourceLocationArgument.getId(context, "id");
+
+    BookData bookData = BookLoader.getBook(book);
+
+    Path gameDirectory = Minecraft.getInstance().gameDirectory.toPath();
+    Path screenshotDir = Paths.get(gameDirectory.toString(), Screenshot.SCREENSHOT_DIR, "mantle_book", book.getNamespace(), book.getPath());
+
+    if(bookData != null) {
+      int width = BookScreen.PAGE_WIDTH_UNSCALED * 2;
+      int height = BookScreen.PAGE_HEIGHT_UNSCALED;
+      float zFar = 1000.0F + 10000.0F * 3;
+
+      bookData.load();
+      BookScreen screen = new BookScreen(Component.literal("Book"), bookData, "", null, null);
+      screen.init(Minecraft.getInstance(), width, height);
+      screen.drawArrows = false;
+
+      if(!screenshotDir.toFile().mkdirs() && !screenshotDir.toFile().exists()) {
```

Comment:
Probably worth checking this before you "open" the screen, as we don't need it open to validate the folders.